docs/modules/ROOT/pages/references/architecture/sli_reporting.adoc

```diff
@@ -0,0 +1,216 @@
+The querying API is effectively a proxy for the Mimir querying API.
+
+The querying API is capable of providing the following for each customer-facing SLO:
+
+* Adjusted SLI Error Rate: this corresponds to `slo:sli_error:ratio_rate1h` with every data point inside a relevant downtime window set to zero.
+* Adjusted Cumulative SLI Error Rate: This corresponds to a `sum_over_time( .. [32d:1h])` of the above result.
+* Adjusted Cumulative SLO Burndown: This corresponds to 1 minus (the above divided by the corresponding SLO error budget).
+* Adjusted Cluster Availability Percentage: This corresponds to the SLO plus (1-SLO times the above).
+
+For each query, the service returns all the datapoints in the timeseries, at a granularity of 1 per hour.
+
+The Querying API accepts the following query parameters:
+* `from`: Timestamp of the start of the timeframe for which data is delivered
+* `to`: Timestamp of the end of said timeframe
+* `cluster_id`: ID of the cluster for which data is requested
+
+Accepting these as query parameters will simplify eventual integration with Grafana.
+
+Sample query response:
+[source,json]
+----
+{
+  "cluster_id": "c-appuio-cloudscale-lpg-2",
+  "sli_data": {
+    "ingress": {
+      "objective": 0.999,
+      "error_budget": 4,
+      "error_rate_1h": [
+        {
+          "timestamp": "2025-07-08T12:00:00Z",
+          "value": 0
+        },
+        {
+          "timestamp": "2025-07-08T13:00:00Z",
+          "value": 0.1
+        },
+        {
+          "timestamp": "2025-07-08T14:00:00Z",
+          "value": 0.3
+        },
+        {
+          "timestamp": "2025-07-08T15:00:00Z",
+          "value": 0.0
+        }
+      ]
+    },
+    "api": {
+      "objective": 0.999,
+      "error_budget": 1.8,
+      "error_rate_1h": [
+        {
+          "timestamp": "2025-07-08T12:00:00Z",
+          "value": 0
+        },
+        {
+          "timestamp": "2025-07-08T13:00:00Z",
+          "value": 0.4
+        },
+        {
+          "timestamp": "2025-07-08T14:00:00Z",
+          "value": 0.2
+        },
+        {
+          "timestamp": "2025-07-08T15:00:00Z",
+          "value": 0.0
+        }
+      ]
+
+    }
+  }
+}
+----
+
+=== Downtime reporting
+
+Downtime windows can be added manually via the Downtime Window API.
+However, in some cases we can automate the reporting of downtime windows, such as when the 3rd party maintains a status page.
+Generally, such status pages provide an RSS feed with information about downtimes.
+
+RSS feeds are only loosely standardized and it's hard to generalize a field mapping for this specific use of RSS.
+For that reason, it would be significant effort to build our own RSS reading service.
+Instead, RSS ingestion can be handled by a simple bash script that's regularly run in a Kubernetes CronJob on VSHN's central cluster.
+
+We can use multiple bash scripts for different sources of RSS feeds.
+The bash script sends the parsed outage time window to the Downtime Window API.
```

Comment:
Is this good enough? I wasn't sure how many statuspages we'd be monitoring this way, I'm assuming it's only a small handful.

RSS is pretty cursed to work with; not looking forward to this part.

It is possible that some general-purpose RSS transformer already exists, which we can use. (I have yet to check)